api/openapi_server/controllers/service_provider_controller.py

```diff
@@ -1,15 +1,12 @@
-
+# Third Party
 import connexion
-import six
 import traceback
 
-from openapi_server.models.api_response import ApiResponse  # noqa: E501
+# Local
 from openapi_server.models.service_provider import ServiceProvider  # noqa: E501
-from openapi_server.models.service_provider_with_id import ServiceProviderWithId
-from openapi_server.models import database as db
-from sqlalchemy.orm import Session
+from openapi_server.repositories.service_provider_repository import HousingProviderRepository
 
-db_engine = db.DataAccessLayer.get_engine()
+housing_provider_repository = HousingProviderRepository()
```

Comment:
would like to do dependency injection with this using the lib `dependency_injector` but need to setup containers. figured i'd save that for a separate PR.